e2e/fixtures/create-pages/src/server-entry.tsx

```diff
@@ -344,14 +345,19 @@ const pages: ReturnType<typeof createPages> = createPages(
     }),
 
     createPage({
-      render: 'static',
+      render: 'dynamic',
       path: '/slices',
-      slices: ['slice001', 'slice002'],
+      slices: ['slice001', 'slice002', 'slice003'],
```

Comment:
lazy slice doesn't need to be sent with the page. if it's sent, it's not lazy.

```suggestion
      slices: ['slice001', 'slice002'],
```

Reply:
Oooo gotcha. Thanks for the explanation.